spec/spec_helper.rb

```diff
@@ -1,5 +1,12 @@
 require "rspec"
 
+# To help produce better bug reports in Rubinius
+if RUBY_ENGINE == "rbx"
+  $DEBUG = true # would be nice if this didn't fail ... :(
+  require 'rspec/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.